tests/test_logs/files/typescript/buffer.ts

```diff
@@ -0,0 +1,520 @@
+import type SourceMap from "./source-map.ts";
```

Comment:
I like to include the details of the original source in the commit when vendoring code for a fixture. I think it comes from here?
https://github.com/babel/babel/blob/ca998707a1c89da04aa849df9c340e06ce6da955/packages/babel-generator/src/buffer.ts

Reply:
Makes sense - did you mean adding a comment or something at the top of the file? Like
```
// Code from https://github.com/babel/babel/blob/ca998707a1c89da04aa849df9c340e06ce6da955/packages/babel-generator/src/buffer.ts
```

Reply:
I would normally just include it in the commit message where we add the file.